wifiphisher/common/interfaces.py

```diff
@@ -160,6 +181,8 @@ def __init__(self, name, card_obj, mac_address):
         self._has_ap_mode = False
         self._has_monitor_mode = False
         self._is_wireless = False
+        self._is_managed_by_nm = False
```

Comment:
But this side-effect will cause the interface down, users will experience the internet access is not available. We should prevent this happen, right?